local_planner/src/nodes/local_planner_node.cpp

```diff
@@ -18,7 +18,7 @@ namespace avoidance {
 LocalPlannerNode::LocalPlannerNode(const ros::NodeHandle& nh,
                                    const ros::NodeHandle& nh_private,
                                    const bool tf_spin_thread)
-    : nh_(nh), nh_private_(nh_private) {
+    : nh_(nh), nh_private_(nh_private), spin_dt_(0.1) {
```

Comment:
100ms? So is 10fps enough, the camera fps is 30fps.

Reply:
It was 10fps before so first, it was not appropriate to change it in this PR. Second, replanning rate doesn't necessarily result in safer or smoother trajectories as the dynamics along the entire replanned trajectory is not that fast.